### DISCUSSION TITLE
UpdateHub v2.3.0

### DISCUSSION THREAD

**Discussion by onyx-nxt** [NexovaDev/UpdateHub#422] [Announcements]
## Version
`2.3.0`

# Changelog
- Finally fixed the timeout error, like actually this time
- Fixed debug info string not including some info
- New update parser, detect even more apps
- Security improvements
- Critical error handling improvements
- Fixed MS Store icon source crashing app
- Reduced possibilities for errors
- Updated app engine
- New setting for the new feature
- Improved performance slightly
- Fix misleading dark mode toggle (#375)
- And more bug fixes

> [!NOTE]
> In order to use the new parser, you must have [PowerShell 7 or later](https://github.com/PowerShell/PowerShell/releases/latest "Latest PowerShell release") installed. Once installed, restart UpdateHub via the system tray. You will get a popup after installation that says `Installing additional dependencies...`. This is because instead of using WinGet CLI to fetch outdated apps, and then using an unreliable RegExp to parse that data which would sometimes cause a lot of errors, we now use the PowerShell module. However, it is not bundled with our app and is not pre-installed, so an additional download is required. You can still use UpdateHub with the old parser.

# How to update
Click the Update Available button in the sidebar. (if you don't see it, force close the app by pressing the shift key, clicking the X icon in the windows title bar, and then relaunch the app.)
<img src="https://user-images.githubusercontent.com/71038229/229697691-43ef66a0-a29d-4e06-af4b-4df0ab5eb772.png" width="572" />
Then wait for it to download the update file, and it will install automatically. You now have the latest version!

### VirusTotal scan
This app is not a virus, trust me. If you want, here's a [VirusTotal scan](https://www.virustotal.com/gui/file/e4a2dc8ec5ebba562234e2e4061d03f754aeaacf8f71a6ee6d42cf39b89f870c/ "VirusTotal Scan"). As you can see, no anti-virus software has flagged it as a virus. The executable below's SHA-256 Hash is `E4A2DC8EC5EBBA562234E2E4061D03F754AEAACF8F71A6EE6D42CF39B89F870C`, which matches the VirusTotal scan.

### Remember, if you encounter any bugs in UpdateHub, report them so we can fix them!

# Thanks for downloading!

### SHA256
`E4A2DC8EC5EBBA562234E2E4061D03F754AEAACF8F71A6EE6D42CF39B89F870C`

<hr /><em>This discussion was created from the release <a href='https://github.com/NexovaDev/UpdateHub/releases/tag/v2.3.0'>UpdateHub v2.3.0</a>.</em>

**Comment by abcpop** [comment]
[UpdateHub v2.3.0] claim to start as admin for update of windows / MSStore even started in admin mode
It was not the case with previous version [UpdateHub v2.2.3]

**Reply by onyx-nxt** [reply]
Going to fix this soon, thanks for reporting

**Reply by onyx-nxt** [reply]
Sorry for the late release, can you update to [v2.3.1](https://github.com/NexovaDev/UpdateHub/releases/tag/v2.3.1) and try again?

**Reply by abcpop** [reply]
Thank you. 
2.3.1  fixed the admin request.
Best regards
